hive-metastore/src/main/java/org/apache/iceberg/hive/MetastoreUtil.java

```diff
@@ -48,14 +50,28 @@ public class MetastoreUtil {
   private MetastoreUtil() {}
 
   /**
-   * Calls alter_table method using the metastore client. If possible, an environmental context will
-   * be used that turns off stats updates to avoid recursive listing.
+   * Calls alter_table method using the metastore client. If the HMS supports then, environmental
```

Comment:
How about "If the HMS supports then" -> "If the HMS supports it"

Reply:
Done.
Thanks!